repairs/public/js/work_order.js

```diff
@@ -1,4 +1,4 @@
-erpnext.production_order.make_se = function (frm, purpose) {
+erpnext.work_order.make_se = function (frm, purpose) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['erpnext' is not defined. (no-undef)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)